lib/src/Pos/Web/Types.hs

```diff
@@ -9,12 +9,17 @@ module Pos.Web.Types
 import           Universum
 
 -- | Stages of SSC.
+-- also called GodTossing algorithm.
+-- GodTossing is a coin tossing with guaranteed output delivery. Nodes exchange commitments, openings, and shares, and in the end arrive at a shared seed.
+-- See https://eprint.iacr.org/2016/889.pdf (“A Provably Secure Proof-of-Stake Blockchain Protocol”), section 4 for more details.
```

Comment:
Please wrap text so that its width is at most 80 characters.

Reply:
ok